.github/workflows/python-build-and-lint.yml

```diff
@@ -37,16 +37,18 @@ jobs:
           path: python/dist/
 
       - name: Build Documentation
-        uses: ammaraskar/sphinx-action@0.4
-        with:
-          pre-build-command: "pip install aac"
-          docs-folder: "docs/"
+        run: |
+          cp -R ../docs/ ./docs
+          cd ./docs
```

Comment:
@lizzcondrey, I don't think I phrased my questions correctly because those responses do not answer what I was asking. Here's an attempt to phrase with more detail/context what I actually want to know.

I realize the changes in this file relate to what you talked about yesterday regarding using the old way of building the docs but it isn't clear to me  why we're copying the `docs/` directory instead of just running the `sphinx-build` command in the docs directory we already have. If the answer is: "because that's how it was done previously" then my follow up question is: "for what reason do we keep it like that?"

Regarding my question about the distinction between this `sphinx.yaml`, I was not talking about uploading anything but if you notice the "Build Documentation" stage in this file and the one in `sphinx.yaml` the one here copies the `docs/` directory and the one in `sphinx.yaml` does not. If we must do it in the "Build Documentation" stage in this file and not in the "Build Documentation" stage in the `sphinx.yaml` file, why? What's the difference?

Reply:
Okay, think I follow. 


I had tried just building within `docs/` directory and uploading the generated output in the next step. You had seen some of the runs where I was trying to find the previously generated output, but it wasn't where I was expecting it to be. 

Copying it into `python\docs` and building let it exist where I expect to leverage for the upload in the next step.

Three for additional solutions, could move docs to within the `python/` directory to begin with or could build within docs and copy the built pages instead then upload. 




For the `sphinx.yaml`, the action allowing upload operates differently and can go up the level to pull from the original `docs/` directory based on how the flows operated previously with the `sphinx-action`. I could be wrong and there may have been moving around that happened within the `sphinx-action` that was used previously that is making us have to go back to moving the location ourselves and need incorporated within the `sphinx.yaml`. I cannot test the workflow from within the branch to evaluate fully, hence just splitting out correcting it if needed and leaving the basic changes of swapping to using our `sphinx-build` command and adjusting the upload path.

Reply:
Okay, that still doesn't track completely for me but we can move on.